Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -1788,9 +1788,6 @@ protected function convertParameterReflectionToArray(ParameterReflection $parame
         if ($parameter->isPassedByReference()) {
             $parameterInformation[self::DATA_PARAMETER_BY_REFERENCE] = true;
         }
-        if ($parameter->isArray()) {
```

Comment:
`isArray()` is deprecated in PHP 8 - we should just handle array like any other builtin type